pandas/_libs/tslibs/period.pyi

```diff
@@ -33,7 +34,7 @@ def get_period_field_arr(
 ) -> npt.NDArray[np.int64]: ...
 def from_ordinals(
     values: npt.NDArray[np.int64],  # const int64_t[:]
-    freq: Frequency,
+    freq: timedelta | BaseOffset | str | None,
```

Comment:
i think None would raise

Reply:
Yes, you are right!

Reply:
Technically, it could be reached with None.
https://github.com/pandas-dev/pandas/blob/9cc98a064e2873ff5ddab62efdb23060c2d52167/pandas/core/arrays/period.py#L965
according to the doc-string freq cannot be `None` but it is typed to also include `None`.

Reply:
Removed `None` from `from_ordinals` and added a mypy ignore comment.